site/src/pages/TemplateVersionEditorPage/MonacoEditor.tsx

```diff
@@ -30,6 +31,12 @@ export const MonacoEditor: FC<{
     };
   }, [editor]);
 
+  useEffect(() => {
+    if (path?.endsWith(".tf")) {
+      registerHCLCompletionProvider(monaco);
```

Comment:
You can register this directly without having to do this since 100% of the time the template uses terraform and the register is very quick. You could do that right after the `loader.config({ monaco });` in line `8`